src/docs/development/ios-14.md

```diff
@@ -32,8 +33,9 @@ see [Issue 60634]({{site.github}}/flutter/flutter/issues/60634).
 ## Launching the app from the home screen
```

Comment:
My notes are a superset of the existing entries there. We can remove this one now (it's a subset of "Launching debug Flutter without a host computer", which is more exact since it's not just home screen).